flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/hint/StateTtlHint.java

```diff
@@ -92,6 +96,32 @@ public static Map<Integer, Long> getStateTtlFromHintOnBiRel(List<RelHint> hints)
         return stateTtlFromHint;
     }
 
+    /**
+     * Get the state ttl from hints from the listOptions inside the STATE_TTL {@link RelHint} if
+     * present. Else if returns an empty map. Used for nodes with multiple inputs such as {@link
+     * StreamPhysicalMultiJoin}.
+     *
+     * @return The key of the map is the input side. The value of the map is the state ttl in
```

Comment:
what does "The key of the map is the input side" mean?

nit: the return should start by saying it returns a map before talking about the key and value content.